File: C/Colleen/Colleen.c
#include <stdio.h>

// outside comment

char	*colleen()
{
	return ("#include <stdio.h>%1$c%1$c// outside comment%1$c%1$cchar	*colleen()%1$c{%1$c	return (%2$c%3$s%2$c);%1$c}%1$c%1$cint	main()%1$c{%1$c	// inside comment%1$c	char	*str;%1$c%1$c	str = colleen();%1$c	printf(str, 10, 34, str);%1$c	return (0);%1$c}");
}

int	main()
{
	// inside comment
	char	*str;

	str = colleen();
	printf(str, 10, 34, str);
	return (0);
}

File: Python/Colleen/Colleen.py
# outside comment

def colleen() -> str:
	return "# outside comment{0}{0}def colleen() -> str:{0}	return {1}{2}{1}{0}{0}def main() -> int:{0}	# inside comment{0}	strr: str = colleen(){0}	print(strr.format(chr(10), chr(34), strr), end={1}{1}){0}	return (0){0}{0}if __name__ == {1}__main__{1}:{0}	main()"

def main() -> int:
	# inside comment
	strr: str = colleen()
	print(strr.format(chr(10), chr(34), strr), end="")
	return (0)

if __name__ == "__main__":
	main()